.nvmrc

```diff
@@ -0,0 +1 @@
+12
```

Comment:
Based on my best guess, because the latest Node version gave me big diffs in the package-lock file that I didn't want to include in this PR.